schema-extended/RelatedResource.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Related Resource",
+  "description": "Author-provided identifier (URI or DOI) for a related resource.",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "identifier": {
+      "oneOf": [{
+          "type": "string",
+          "format": "uri"
+        },
+        {
+          "$ref": "Common.json#/definitions/doi"
+        }
+      ]
```

Comment:
I'd recommend against simply overloading the `identifier` to mean `url or doi`, as it's often very useful to know what kind of identifier you're dealing with (for example, if you display this somewhere you'll need to turn the DOI into a URL).

Reply:
That's a good point. I'll revise--thanks!